packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -923,7 +969,7 @@ describe('Checkout', () => {
             });
 
             describe('when `getCheckout` request fails', () => {
-                it('should emit failure event', async () => {
+                it('should emit failure event and set `hasCheckoutLoadedSuccessfully` to `false`', async () => {
```

Comment:
"and" in a test name usually indicates it should be multiple test cases

Reply:
I personally don't agree with this. An `and` in this case indicates an additional `expect`, not a different test case.
I think having more than one `expect` statement per test is perfectly fine, although I know some people prefer not to have them.

I'd agree with your comment in a case in which I had something like the following `when A or B, then C`. That `or` probably means there should be two separate tests.

Reply:
It comes down to personal preference. I won't let it block the PR though because it's not a particularly bad case of using too many `expect`s.